builder/ecs/run_config.go

```diff
@@ -135,6 +135,8 @@ type RunConfig struct {
 	// the ECS created through private ip instead of allocating a public ip or an
 	// EIP. The default value is false.
 	SSHPrivateIp bool `mapstructure:"ssh_private_ip" required:"false"`
+	//If this value is true, packer will skip create image
```

Comment:
nit: we can change the text to call out that Packer will not create an image for the build. 
```suggestion
	//If  true, Packer will not create a final image. Defaults to `false`. 
```

Reply:
Fixed